### ISSUE TITLE
Update json-schema-validator in swagger-compat-spec-parser

### ISSUE THREAD

**Issue by msymons** [swagger-api/swagger-parser#463]
Update swagger-compat-spec-parser to use json-schema-validator v2.2.8 to address CVSS 3.0 level 5.4 security threat which originates from libphonenumber dependency.

Note that json-schema-validator has had a change of groudId from com.github.fge to com.github.java-json-tools.

The libphonenumber transitive dependency results  in a security alert from Nexus IQ OSS security scanning software.   There is no CVE ID.  Just a Sonatype problem code:

sonatype-2015-0090 - libphonenumber - A Cross Site Scripting vulnerability was found which is exploitable by manipulating the inputs.  Reference:

https://github.com/googlei18n/libphonenumber/pull/934

**Comment by msymons** [comment]
Reference PR #466

**Comment by ewaostrowska** [comment]
Yes, this is resolved by https://github.com/swagger-api/swagger-parser/pull/466 and already released. I am closing this issue. Thanks @msymons and @Philzen
